sources/models/README.md

```diff
@@ -72,6 +72,16 @@ The `#[model]` attribute on Settings and its sub-structs reduces duplication and
 * [Model](src/aws-k8s-1.26-nvidia/mod.rs)
 * [Default settings](src/aws-k8s-1.26-nvidia/defaults.d/)
 
+### aws-k8s-1.27: Kubernetes 1.27
+
+* [Model](src/aws-k8s-1.26/mod.rs)
+* [Default settings](src/aws-k8s-1.26/defaults.d/)
+
+### aws-k8s-1.27-nvidia: Kubernetes 1.27 NVIDIA
+
+* [Model](src/aws-k8s-1.26-nvidia/mod.rs)
+* [Default settings](src/aws-k8s-1.26-nvidia/defaults.d/)
```

Comment:
should be 1.27.

Reply:
Odd, none of the changes from `lib.rs` ended up here. Ran a cargo build directly from this directory and now it worked. Changes staged and will push up with next update.